src/components/program/ProgramCourses.jsx

```diff
@@ -56,7 +57,7 @@ const ProgramCourses = () => {
                     (courseRun?.pacingType === PROGRAM_PACING_MAP.INSTRUCTOR_PACED && courseRun.start)
                     && (
                       <div className="course-card-result mb-2">
-                        <FontAwesomeIcon icon={faCalendarAlt} className="fa-calendar-alt mr-2" />
+                        <CalendarMonth className="calendar-icon mr-2" />
```

Comment:
nit: looks like this file is mixing its use between the `Icon` component versus the import icon itself as the component (e.g., `CalendarMonth`). I feel we should try to be consistent (at least when in the same file) in how we use icons.